governance.md

```diff
@@ -26,45 +25,45 @@ Furthermore, the Prometheus Operator community strives for kindness, giving feed
 Team member status may be given to those who have made ongoing contributions to the Prometheus Operator project for at least 3 months.
 This is usually in the form of code improvements, pull-request reviews, issue triaging or notable work on documentation, but organizing events or user support could also be taken into account.
 
-New members may be proposed by any existing Maintainer by email to [prometheus-operator-team](https://groups.google.com/forum/#!forum/prometheus-operator-team). It is highly desirable to reach consensus about acceptance of a new member.
+New members may be proposed by any existing Maintainer by email to the [prometheus-operator mailing list](https://groups.google.com/forum/#!forum/prometheus-operator-team). It is highly desirable to reach consensus about acceptance of a new member.
 However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote) of Team Maintainers.
 
 If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership.
-This email will also be CC'd to [prometheus-operator-team](https://groups.google.com/forum/#!forum/prometheus-operator-team) for record-keeping purposes.
+This email will also be CC'd to the [prometheus-operator mailing list](https://groups.google.com/forum/#!forum/prometheus-operator-team) for record-keeping purposes.
 
 If they choose to accept, the following steps are taken:
 
-* Maintainer is added to the [GitHub organization](https://github.com/prometheus-operator) as *Owner*.
-* Maintainer is added to the [prometheus-operator-team](https://groups.google.com/forum/#!forum/prometheus-operator-team).
-* Maintainer is added to the list of team members [here](MAINTAINERS.md)
-* New maintainer is announced on the [Prometheus Operator Twitter](https://twitter.com/PromOperator) by an existing team member.
+* The maintainer is added to the [prometheus-operator organization](https://github.com/prometheus-operator) with the `Owner` role.
+* The maintainer is added to the [prometheus-operator mailing list](https://groups.google.com/forum/#!forum/prometheus-operator-team).
+* The maintainer is added to the [MAINTAINERS.md](MAINTAINERS.md) file.
+* The new maintainer is announced on the [Prometheus Operator Twitter](https://twitter.com/PromOperator) by an existing team member.
```

Comment:
Any reason why only the last bullet point has "new" at the beginning? :P

Reply:
good call! no reason except that i didn't notice the discrepancy :)